scripts/commands/changesjob.lua

```diff
@@ -23,23 +23,23 @@ function onTrigger(player, jobId, level)
         return
     end
     jobId = tonumber(jobId) or xi.job[string.upper(jobId)]
-    if (jobId == nil or jobId <= 0 or jobId >= xi.MAX_JOB_TYPE) then
+    if (jobId == nil or jobId < 0 or jobId >= xi.MAX_JOB_TYPE) then
```

Comment:
no parenthesis on condition statements unless needed for order of operations, and please follow style guide for conditionals for more than one condition statement:

```lua

if
    thing == 1 or
    thing ~= 2 and
    anotherThing <= 5
then

```

There are a few other cases in this lua of parenthesis as well

Reply:
Sure, should I update the whole file, just the lines I added/edited, or follow the style of the file? I just followed the style of the surrounding code to minimize the diff to what actually changed.

Reply:
Here is the style guide:
https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md#style-guide

You should always follow it on code you change. Alot of stuff is still remnants of old code formats that have yet to be converted over. But if you code it please follow the guide.